src/form.js

```diff
@@ -185,6 +185,12 @@ define([
         removeHashtag: function(hashtag) {
             delete this.hashtagDictionary[hashtag];
         },
+        clearNullHashtags: function () {
+            this.hashtagDictionary = _.chain(this.hashtagDictionary)
```

Comment:
Someday underscore will release a `filterObject` method so this can be one function call instead of four. But not today.